src/imu/imu_interface.hpp

```diff
@@ -87,6 +87,8 @@ class InertialMeasurementInterface {
    */
   virtual auto GetTemperature(void) noexcept -> int = 0;
 
+  virtual auto SetImuSeam(std::unique_ptr<imu::GenericInertialMeasurementUnit> imu) noexcept -> void = 0;
```

Comment:
No documentation although it is part of the public interface

Reply:
Added documentation here: https://github.com/Squiddrone/Squiddrone/pull/269/commits/a6809cdad043473167d102e6c33a65f24ad24c2a
I think we should basically discuss whether public documentation for self-explanatory interfaces (this is not such an interface) is really necessary. Uncle Bob says it is not necessary if the name is well chosen.